easysid.cpp

```diff
@@ -38,6 +55,35 @@ EXPORT void end_capture()
     }
 }
 
+EXPORT void set_chip(uint32_t chip)
+{
+    if (chip < 0 || chip > 1) {
+        // TODO: Implement errors, or SEH to handle these scenarios.
+        return;
+    }
+    sid.set_chip_model(static_cast<chip_model>(chip));
+}
+
+EXPORT void set_sample_parameters(double clock, uint32_t sample_mode, uint32_t sample_freq)
+{
+    if (sample_mode < 0 || sample_mode > 3) {
+        // TODO: Implement errors, or SEH to handle these scenarios.
+        return;
+    }
+    if (sample_freq < 0 || sample_freq > 48000) {
+        // TODO: Implement error checking...
+        return;    
+    }
+    g_sample_rate = sample_freq;
+    if (sid.set_sampling_parameters(clock, static_cast<sampling_method>(sample_mode), static_cast<double>(sample_freq))) {
```

Comment:
What happens to the existing easysid output objects when the sampling rate is changed after they have been opened? That case is not handled.